### PR TITLE
Email validation

### PR DESCRIPTION
**Pull request by lucbelliveau**
Take 2.

This version takes it's input from the database to allow for easier configuration, and provides a default pattern of .*, effectively disabling it.

### CODE REVIEW

**Comment by lucbelliveau** [comment]
@bartjkdp this supersedes my previous PR...

**Comment by bartjkdp** [comment]
Thanks a lot, this looks great. Really like the approach of a separate app. Do people of GCTools already use it / reviewed it? I totally am fine with this approach, as far as I am concerned we could merge this in. Final review and merge by people of GCTools?

**Comment by bartjkdp** [comment]
I added @ToferC and @markwoof as project admins, so they can review this pull request and merge it in.

File: emailvalidator/migrations/0001_initial.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('regex', models.CharField(max_length=100)),
-                ('group', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='emailvalidator.EmailDomainGroup')),
+                ('EmailDomainGroup', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='emailvalidator.EmailDomainGroup')),
```
Comment (bartjkdp) [inline emailvalidator/migrations/0001_initial.py]: Just one very small style thing. Think general Django-style is to use lowercase & underscore for field-names. For example:

    ('accepted_terms', models.BooleanField(default=False)),

in https://github.com/Pleio/pleio_account/blob/master/core/migrations/0001_initial.py#L25

Comment (lucbelliveau) [reply emailvalidator/migrations/0001_initial.py]: Good catch, fixed.

**Pull request merged**